app/controllers/publish/courses_controller.rb

```diff
@@ -24,6 +24,11 @@ def show
       flash.delete(:error_summary)
     end
 
+    def placements
```

Comment:
How do we feel about introducing a new controller for Placements and using a `#show` action probably a `resource :placements` in the routes?

I know the Publish codebase has a number of "non-RESTful" actions and we don't need to add to them 🤞 if we don't have to.

Reply:
I've put the actions in their own controller(index action). I'm not sure about using resources, the way the routes are setup makes it difficult.

On find we need these two params `/course/:provider_code/:course_code/placements`
And on publish I have to use `/placements` rather than `/school-placements` as the path is already taken, the controller is called `school-placements`, it's an existing controller.